source/tyr/tyr/binarisation.py

```diff
@@ -71,17 +71,42 @@ def unzip_if_needed(filename):
     return working_directory
 
 
-def move_to_backupdirectory(filename, working_directory):
+def manage_file_with_unwanted_char(file_basename, sub_string):
```

Comment:
what happens if file's name is `myfile(1).osm(2).pbf`?

Reply:
It will rename as myfile_xxxx.pbf where xxx = formatted_datatime